report_barcode_elaphe/models/report.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 bloopark systems (<http://bloopark.de>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import models
+from openerp.tools.safe_eval import safe_eval as eval
```

Comment:
I would prefer this left as the name `safe_eval`. Overriding namespaces you don't own isn't good IMO & when I saw it later in the code I had to jump back up to the imports to see if it was being done right. Best to leave it explicit